lib/ansible/modules/crypto/luks_device.py

```diff
@@ -249,10 +254,15 @@ def is_luks(self, device):
         result = self._run_command([self._cryptsetup_bin, 'isLuks', device])
         return result[RETURN_CODE] == 0
 
-    def run_luks_create(self, device, keyfile):
+    def run_luks_create(self, device, keyfile, keysize):
         # create a new luks container; use batch mode to auto confirm
-        result = self._run_command([self._cryptsetup_bin, 'luksFormat',
-                                    '-q', device, keyfile])
+        if keysize is not None:
```

Comment:
How about putting the command into a list, and extending that list if `keysize is not None`, and calling `self._run_command()` later?